packages/meditrak-app/app/api/TupaiaApi.js

```diff
@@ -22,7 +22,7 @@ const SOCIAL_FEED_ENDPOINT = 'socialFeed';
 const CURRENT_USER_REWARDS_ENDPOINT = 'me/rewards';
 const DEV_BASE_URL = 'https://dev-api.tupaia.org/v2';
 const PRODUCTION_BASE_URL = `https://${isBeta ? `${betaBranch}-` : ''}api.tupaia.org/v2`;
-export const BASE_URL = centralApiUrl ?? (__DEV__ ? DEV_BASE_URL : PRODUCTION_BASE_URL);
+export const BASE_URL = __DEV__ ? centralApiUrl || DEV_BASE_URL : PRODUCTION_BASE_URL;
```

Comment:
TIL `a ?? b` only returns a if a is `null` or `undefined`, not falsy like most other conditionals in javascript. 

```
'' ? '' : 'x'   --> 'x'
'' || 'x'       --> 'x'
'' ?? 'x'       --> ''
```

I've also moved centralApiUrl to behind the __DEV__ check so it can only be used in local development.

Reply:
Ah, nullish vs falsey, a classic gotcha